odk1-src/central-projects.rst

```diff
@@ -130,12 +130,20 @@ To manage App Users for a Project, you can navigate to the Project overview page
 Archiving a Project
 ~~~~~~~~~~~~~~~~~~~
 
-When you Archive a Project, the following things will happen:
+When you Archive a Project, the following things become frozen:
+
+ - Any Forms that are still in :guilabel:`Open` or :guilabel:`Closing` states will remain in those states.
+ - Any users who are Project Managers will retain their access.
```

Comment:
I'd be explicit here about what happens to App Users.

Reply:
agreed; will do.